src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetUnallocatedCasesByTeamTests.kt

```diff
@@ -154,6 +163,11 @@ class GetUnallocatedCasesByTeamTests : IntegrationTestBase() {
         Triple("ZZZZZZZ", true, true),
```

Comment:
Update  Get unallocated cases by team where probation-estate API is successful and does not return LAO cases to change LAO to restricted

Reply:
done